source/getting-started/use-cases/vulnerability-detection.rst

```diff
@@ -1,93 +1,140 @@
   
 Vulnerability detection
 =======================
 
-Vulnerable software applications are commonly targeted by attackers to compromise endpoints and gain a persistent presence on targeted networks. Relying on its software inventory capabilities, the Wazuh platform maintains an up-to-date list of all applications installed on the endpoints where the :doc:`Wazuh agent <../components/wazuh-agent>` is located. 
+Software vulnerabilities are weaknesses in code that can allow attackers to gain access to or manipulate the behavior of an application. Vulnerable software applications are commonly targeted by attackers to compromise endpoints and gain a persistent presence on targeted networks.
 
-In addition, Wazuh identifies vulnerable applications and produces risk reports by correlating that list with the National Vulnerability Database (NVD) and with information collected from different operating system vendors. To detect vulnerable software, Wazuh uses a database of Common Vulnerabilities and Exposures (CVE) created automatically by processing data pulled from the following sources:
+Vulnerability detection is the process of identifying these flaws before they are discovered and exploited by attackers. The goal of vulnerability detection is to identify vulnerabilities so that remediation can be carried out to prevent successful attacks.
 
-- `CVEs for Ubuntu Linux distributions <https://canonical.com>`_
-- `CVEs for Red Hat and CentOS Linux distributions <https://access.redhat.com>`_
-- `CVEs for Debian Linux distributions <https://www.debian.org>`_
-- `CVEs for Arch Linux distributions <https://security.archlinux.org>`_
-- `CVEs for Amazon Linux distributions <https://alas.aws.amazon.com/>`_
-- `CVEs for SUSE Linux distributions <https://www.suse.com/support/security/oval/>`_
-- `CVEs from the National Vulnerability Database <https://nvd.nist.gov/>`_
-- `Microsoft Security Response Center <https://www.microsoft.com/msrc>`_
+The :doc:`Wazuh agent </getting-started/components/wazuh-agent>` collects the software inventory data from a monitored endpoint periodically and sends it to the Wazuh server. The Wazuh :doc:`Vulnerability Detector </user-manual/capabilities/vulnerability-detection/index>` module correlates the software inventory data to vulnerability feeds to detect vulnerable software on a monitored endpoint. Wazuh identifies vulnerable applications and produces risk reports using the information collected from the different operating system vendors and vulnerability databases. The :doc:`Vulnerability Detector </user-manual/capabilities/vulnerability-detection/how-it-works>` module uses a database of Common Vulnerabilities and Exposures (CVEs) created automatically by processing data pulled from several sources including :doc:`Wazuh feeds </user-manual/capabilities/vulnerability-detection/how-it-works>`.
```

Comment:
The link to the "Vulnerability detection - How it works" section appears twice in the same paragraph;  consider removing one.

Reply:
Fixed.